src/op_analytics/datapipeline/models/templates/daily_address_summary.sql.j2

```diff
@@ -82,11 +83,29 @@ SELECT
   ) AS l1_base_price_avg_gwei,
 
   wei_to_gwei(safe_div(sum(l1_blob_fee), sum(l1_blob_scaled_size)))
-    AS l1_blob_fee_avg_gwei
+    AS l1_blob_fee_avg_gwei,
+
+  -- Data Processed
+  sum(input_zero_bytes) AS input_zero_bytes_sum,
+  sum(if(success, input_zero_bytes, 0)) AS success_input_zero_bytes_sum,
+
+  sum(input_nonzero_bytes) AS input_nonzero_bytes_sum,
+  sum(if(success, input_nonzero_bytes, 0)) AS success_input_nonzero_bytes_sum,
+
+  sum(input_byte_length) AS input_byte_length_sum,
+  sum(if(success, input_byte_length, 0)) AS success_input_byte_length_sum,
+
+  sum(estimated_size) AS estimated_size_sum,
+  sum(if(success, estimated_size, 0)) AS success_estimated_size_sum
+
 FROM
-  transaction_fees
+  refined_transactions_fees
+WHERE
+  NOT is_system_transaction
+  AND gas_price > 0
```

Comment:
this used to be at the transactions_fees level, but I unfiltered there(so we preserve those transactions) and brought it here.